### PR TITLE
Add two epic digraphs

### LINKED ISSUE #375: Add an element to empty the global array
pls I need to beat APL

(created by lyxal [here](https://chat.stackexchange.com/transcript/message/59855112))

### LINKED ISSUE #453: Every slice of a string/list digraph
`(a: num, b: str | lst) => [b[0::a],b[1::a]...b[a-1::a]]` or something like that - get all slices of a list, skipping a certain number of items

### PR DESCRIPTION
**Pull request by Lyxal**
Closes #375 
Closes #453

### CODE REVIEW

**Review by PyGamer0 — APPROVED** [review]
epic indeed

**Comment by chunkybanana** [comment]
Did you read #453?  That's not what I meant.

> `(a: num, b: str | lst) -> [b[0::a],b[1::a]...b[a-1::a]]`

File: documents/knowledge/elements.yaml [hunk 1]
```diff
@@ -4488,6 +4488,24 @@
   tests:
     - "[[1,5,3],[4,2,6]] : [1, 2, 3]"
 
+- element: "Þs"
+  name: All Slices of a List
```
Comment (pxeger) [inline documents/knowledge/elements.yaml:4492]: I feel like this would make a good overload of a one-byter. Are there any `lst-int`s available?

**Review by pxeger — APPROVED** [review]
generally LGTM though

**Pull request merged**